packages/react/src/components/DropdownMenu/DropdownMenuHeading.tsx

```diff
@@ -0,0 +1,25 @@
+import cl from 'clsx/lite';
+import { type HTMLAttributes, forwardRef, useContext } from 'react';
+import { Paragraph } from '../Typography';
+import { DropdownMenuCtx } from './DropdownMenuContext';
+
+export type DropdownMenuHeadingProps = HTMLAttributes<HTMLHeadingElement>;
+
+export const DropdownMenuHeading = forwardRef<
+  HTMLHeadingElement,
+  DropdownMenuHeadingProps
+>(function DropdownMenuHeading({ children, className, ...rest }, ref) {
+  const { size } = useContext(DropdownMenuCtx);
+
+  return (
+    <Paragraph asChild size={size}>
```

Comment:
Er litt rart å ha en Heading, men at sizes følger Paragraph - at det ikke matcher med Heading-hierarkiet man kjenner igjen i Figma, men dette kan kanskje bli en egen oppgave.
Også litt rart å ikke kunne sette `level`. Tenker - dersom vi kaller noe `*.Heading`, burde det extende `Heading`-komponenten

Reply:
Forsåvidt einig, men valgte å ikkje ta denne endringen i denne PRen, då det går også litt på design

Reply:
I created #2439 for this